### ISSUE TITLE
Version 1.4.0 Alpha-5

### ISSUE THREAD

**Issue by gabriele-v** [moneymanagerex/moneymanagerex#1767]
@moneymanagerex/desktop-mmex-team 
Should we release a 1.4.0-alpha.5?

**Comment by vomikan** [comment]
I have marked some ASAP issues as 'good first issue'. There are only two issues left:
https://github.com/moneymanagerex/moneymanagerex/labels/good%20first%20issue
It's not critical. I think we should involve more users for testing. I suggest release beta.

**Comment by slodki** [comment]
Please look at https://github.com/moneymanagerex/moneymanagerex/milestones/v1.4.0
We should prepare updated transaltions and documentation (manual) too!

**Comment by gabriele-v** [comment]
IMHO we should just refine documentation/translations/changelog.
Other items listed in 1.4.0 milestone (filetypes, dataversion, sqlite application id) could be postponed.

Main issue is this one: https://github.com/moneymanagerex/android-money-manager-ex/issues/1234
If we officially release new DB changes, we will break Android compatibility, but even worst, we will have DB Version out of sync because of the highlighted downgrade issue in Android version.

**Comment by slodki** [comment]
There is problem with Mac version too. MMEx is blocked by MacOS by default. Multiple crash issues are reported. Installer package can have wrong description (?). It's still alpha IMHO - maybe beta...

**Comment by slodki** [comment]
@moneymanagerex/desktop-mmex-team
Releasing new desktop version is as simple task as creating new tag in git repo in master branch now. Tag value must follow symver: vN.NN.NNNN with optional -beta.NN or similar suffix. This tag value will be used in all MMEX sources, will create new release at GitHub Releases and publish all binary packages. Users will be notified inside desktop app too.

I'll create -alpha.5 today after merging PR.

**Comment by xraynaud** [comment]
@slodki: I compiled yesterday mmex-alpha6 on my macbook pro (MacOS 10.13) and it works like a charm. I really don't understand what "MMEx is blocked by MacOS by default" means. ALL softwares but those dowloaded from the appstore are blocked by default and you have to go in the security section of the preferences pane to allow installation. This has to be done only once. In older versions (<MacOS X 10.11), I recall you had to allow installation everytime.

**Comment by gabriele-v** [comment]
I think this one could be closed, because we are still undecided how to proceed with database upgrades
